src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -39,28 +53,102 @@ def show_configstore(cmd, client, name, resource_group_name=None):
     return client.get(resource_group_name, name)
 
 
-def configstore_update_get():
-    return ConfigurationStoreUpdateParameters()
-
-
-def configstore_update_set(cmd,
-                           client,
-                           parameters,
-                           name,
-                           resource_group_name=None):
+def update_configstore(cmd,
+                       client,
+                       name,
+                       resource_group_name=None,
+                       tags=None,
+                       sku=None):
     if resource_group_name is None:
         resource_group_name, _ = resolve_resource_group(cmd, name)
 
-    update_params = ConfigurationStoreUpdateParameters(tags=parameters.tags)
+    update_params = ConfigurationStoreUpdateParameters(tags=tags,
+                                                       sku=sku)
+
     return client.update(resource_group_name=resource_group_name,
                          config_store_name=name,
                          config_store_update_parameters=update_params)
 
 
-def configstore_update_custom(instance, tags=None):
-    if tags is not None:
-        instance.tags = tags
-    return instance
+def assign_managed_identity(cmd, client, name, resource_group_name=None, identities=None):
+    if resource_group_name is None:
+        resource_group_name, _ = resolve_resource_group(cmd, name)
+
+    if identities is None:
+        identities = [SYSTEM_ASSIGNED_IDENTITY]
```

Comment:
it looks you have default value for identities. please let users know this using `default=[system]`

Reply:
Added in _params.py.

Reply:
and then can we move the assignment here?

Reply:
In _params.py we actually hard coded the default value as part of the help message. —identities is a list so if we move the default assignment here as the method signature, it will be [[“system”]], which is a little confusing for customers.